scripts/lib/CIME/case/check_input_data.py

```diff
@@ -37,15 +73,16 @@ def _download_if_in_repo(server, input_data_root, rel_path, isdirectory=False):
             return server.getfile(rel_path, full_path)
 
 ###############################################################################
-def check_all_input_data(self, protocol=None, address=None, input_data_root=None, data_list_dir="Buildconf", download=True):
+def check_all_input_data(self, protocol=None, address=None, input_data_root=None, data_list_dir="Buildconf",
+                         download=True, chksum=False):
```

Comment:
Please add documentation for the new chksum argument. (Ideally there would be documentation for all arguments.)

Reply:
This comment and my other similar comment about argument documentation are still unresolved: this and other functions could use more documentation of their arguments.